### DISCUSSION TITLE
Enter record > save > return to list view

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#16798] [How to]
While entering manually lots of records it would be very helpful if system diverts user to list view (after save).
Is there a way to do it without touching core?

**Comment by mariuszkrzaczkowski** [comment]
the easiest way is to use quick create, the other way is a parameter in the url that determined the return to the list, you can add a new button in the list with this parameter

**Comment by s2b-git** [comment]
Marius, what paremeter should be added to the URL to return to list view? Also interested...
Can you drop sample url? Quikcreate not showing url

**Reply by mariuszkrzaczkowski** [reply]
I don't remember, I'd have to look for it, see in the Save file
